tests/PSRule.Rules.Azure.Tests/Azure.KeyVault.Tests.ps1

```diff
@@ -339,5 +355,18 @@ Describe 'Azure.KeyVault' -Tag 'KeyVault' {
             $ruleResult[0].TargetName | Should -BeLike '*Resources.Parameters2.json';
             $ruleResult[1].TargetName | Should -BeExactly 'kvstoragediag01';
         }
+
+        It 'Azure.KeyVault.Firewall' {
+            $filteredResult = $result | Where-Object { $_.RuleName -eq 'Azure.KeyVault.Firewall' };
+
+            # Fails
+            $ruleResult = @($filteredResult | Where-Object { $_.Outcome -eq 'Pass' });
+            $ruleResult | Should -BeNullOrEmpty;
+#            test should be fail when the module will be published            
+#            $ruleResult = @($filteredResult | Where-Object { $_.Outcome -eq 'Fail' });
+#            $ruleResult | Should -Not -BeNullOrEmpty;
+#            $ruleResult.Length | Should -Be 1;
+#            $ruleResult.TargetName | Should -Be 'keyvault-001';
```

Comment:
Since this will fail
```suggestion
            $ruleResult = @($filteredResult | Where-Object { $_.Outcome -eq 'Fail' });
            $ruleResult | Should -Not -BeNullOrEmpty;
            $ruleResult.Length | Should -Be 1;
            $ruleResult.TargetName | Should -Be 'keyvault-001';
            
            # No passes
            $ruleResult = @($filteredResult | Where-Object { $_.Outcome -eq 'Pass' });
            $ruleResult | Should -BeNullOrEmpty;
```